packages/cardano-graphql/src/operations/protocolParameters.graphql

```diff
@@ -0,0 +1,19 @@
+query ProtocolParameters {
+  # Review: I'm wondering if there should be some sort of version or serial # to sort by
```

Comment:
The `protocolVersion.major` + `minor` + `patch` by descending will sort by most recent